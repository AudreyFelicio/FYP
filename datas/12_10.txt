12 10
2 6 3 11 5 9 12 4 8 10 1 7 
12 7 1 9 10 3 11 4 5 8 6 2 
12 3 8 6 9 5 11 7 4 2 1 10 
1 11 12 2 4 9 7 10 3 8 5 6 
7 11 2 12 9 6 3 5 10 4 8 1 
12 3 7 5 6 11 8 1 2 10 4 9 
8 2 11 10 7 4 3 12 5 9 6 1 
2 12 5 4 7 9 1 10 8 11 6 3 
1 5 3 9 8 2 6 12 10 11 7 4 
4 11 9 8 2 3 10 12 6 7 5 1 
