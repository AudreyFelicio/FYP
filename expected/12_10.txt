56
2 11 12 9 3 10 7 4 5 8 6 1 